docs/pipeline.rst

```diff
@@ -173,23 +173,30 @@ default can be set and then can be overridden in the individual job configuratio
 Adding test runs
 ~~~~~~~~~~~~~~~~
 
-Test runs can be added by creating a new properties section in the
-`integration tests script <https://github.com/mozilla/bedrock/blob/master/docker/bin/run_integration_tests.sh>`_
-with the parameters of the new test run. This is simply a bash script and you can duplicate a clause of the case staement.
+Test runs can be added by creating a new job in the `.gitlab-ci.yml file in the www-config repository`_
```

Comment:
This section arguably belongs under https://www-config.readthedocs.io/#testing but for this PR I'm leaving them here to keep the changes in context, with the intention of moving them in a future pair of PRs to www-config and bedrock.